content/en/docs/instrumentation/js/serverless.md

```diff
@@ -173,3 +173,123 @@ function.
 
 You should now be able to view traces produced by OpenTelemetry from your Lambda
 function in the backend!
+
+## GCP function
+
+The following shows how to instrument [http triggered function](https://cloud.google.com/functions/docs/writing/write-http-functions) using the Google Cloud Platform (GCP) UI.
+
+### Creating function
+
+Login to GCP and select the project where your function should be placed. From serverless select Cloud Functions. Select create function and select environment. This example uses second generation. 
```

Comment:
> From serverless select Cloud Functions

What does this mean? Is this a menu option?

"Select Serverless > Cloud Functions"

Same with other examples.

> uses second generation. 

What is second generation? Is there a link that we'd need to have?

Reply:
@klempinen can you clarify what "second generation" is, you need to assume that people might not know this, if necessary a link to the google function docs would be enough.